src/main/java/frc/utils/beamBreaker/BeamBreakerInputs.java

```diff
@@ -0,0 +1,10 @@
+package frc.utils.beambreaker;
```

Comment:
Please rename everything from beam breaker to digitalInput

- IDigitalInputs as the interface
And so on. Beam breaker is a component that is connected via digital input, but outputs exactly  the same as a switch (true and false).